app/controllers/posts_controller.rb

```diff
@@ -1,24 +1,38 @@
 class PostsController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
- [ ] **Required:** Kindly add authorization to this class to ensure that an unauthorized user cannot be given access.

```
class PostsController < ApplicationController
    load_and_authorize_resource
```

Reply:
@brytebee  Thank you very much for your kind review and feedback. I added this to my post controller.
- Did you check the delete buttons, if they are working fine?